### PR TITLE
Add prettier

### PR DESCRIPTION
**Pull request by marileize**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (adilsoncarvalho) [inline .prettierrc:1]: Isn't there a way to configure the source path for prettier to use, so we don't need to say the path when invoking it?

Might be worth checking the documentation imo

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "main": "index.ts",
   "scripts": {
     "build": "tsc",
-    "test": "jest"
+    "test": "jest",
+    "prettier": "prettier --write src"
```
Comment (adilsoncarvalho) [inline package.json]: Maybe prettier can get configured to assume the `src` folder all the time.

Comment (adilsoncarvalho) [inline package.json]: Maybe we could have a check also.

```suggestion
    "prettier": "prettier src",
    "prettier:fix": "prettier --write src"
```

**Review by adilsoncarvalho — CHANGES_REQUESTED** [review]
We might configure `prettier` in a different way, avoiding it to write changes by mistake.

File: .prettierrc (hunk 1 above)
Comment (marileize) [reply .prettierrc:1]: I think I need to change this in the package.json: "prettier:fix": "prettier --write src", but I don't know "how" yet. Just removing "src" doesn't work :D

Comment (adilsoncarvalho) [reply .prettierrc:1]: To remove the path, in this case `src`, we need to add it to prettier configuration, if such an option exists. The documentation only mention that we can [ignore code](https://prettier.io/docs/en/ignore).

Maybe there is nothing we can do, and the paths must be kept in the command line.

Comment (marileize) [reply .prettierrc:1]: It seems that in the command line, we always need to specify the path. Can I finish this conversation?

**Review by adilsoncarvalho — APPROVED** [review]

**Pull request merged**